src/views/blocks/archive-events.php

```diff
@@ -16,9 +16,8 @@
 use Tribe\Events\Views\V2\Template_Bootstrap;
 
 tribe_asset_enqueue_group( Event_Assets::$group_key );
-tribe_asset_enqueue( 'tec-events-iframe-content-resizer' );
 ?>
 
-<div class="tribe-block tribe-block__archive-events">
+<div class="tec-block tec-block__archive-events">
```

Comment:
Careful here - there are styles associated with that first class: https://github.com/the-events-calendar/the-events-calendar/blob/release/B23.maria/src/styles/style.pcss#L6